src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -969,12 +969,19 @@ public void OnRigidityChanged(int desiredRigidity)
 
         if (cost > Editor.MutationPoints)
         {
-            int stepsToCutOff = (int)Math.Ceiling((cost - Editor.MutationPoints) / costPerStep);
-            data.NewRigidity -= (desiredRigidity - previousRigidity > 0 ? 1 : -1) * stepsToCutOff /
-                Constants.MEMBRANE_RIGIDITY_SLIDER_TO_VALUE_RATIO;
+            // TODO: This doesn't work correctly in low editor MP with significantly reduced costs.
+            //       Supposedly unaffordable sliding action makes the tooltip indicate stats changes (even though
+            //       the grabber itself doesn't seem to move) and I can't figure out the maths to fix it - kasterisk
+            //
+            // int stepsToCutOff = (int)Math.Ceiling((cost - Editor.MutationPoints) / costPerStep);
+            // data.NewRigidity -= (desiredRigidity - previousRigidity > 0 ? 1 : -1) * stepsToCutOff /
+            //    Constants.MEMBRANE_RIGIDITY_SLIDER_TO_VALUE_RATIO;
```

Comment:
This code made the slider able to stop at a maximum affordable MP when you jump out of the affordable range.

I can't figure out the maths for this, this always breaks at certain cost multipliers so I commented this out and the glitchy bug is fixed at least.